### ISSUE TITLE
Filters do not support accents

### ISSUE THREAD

**Issue by angel-k** [cgwire/kitsu#712]
On Kitsu 0.12.81
When I try to filter by Assignation with an user who has an accent in their name, the filter gives me 0 results. It seems that the filters do not handle accents.
I have the same problem when a Task Type have an accent.

Example :
The user is called "Léa" and is assigned to tasks.
I search with the filters :
- assignedto=[Léa] -> 0 result
- assignedto=[Lea] -> doesn't filter anything

**Comment by frankrousseau** [comment]
Thank you for reporting!

**Comment by frankrousseau** [comment]
It should be fixed. Can you try again?

**Comment by angel-k** [comment]
I didn't test it before today (Kitsu 0.13.0)
The accents support for Task Type and Assignation seems to work well but now I feel like the "assignedto" filter is buggy...
For some users (with or without accents) it works, for others I get 0 results. It seems to happen when multiple user have the same name (or almost the same). It worked well before.
I guess that the accent support is fixed but that the "assignedto" filter not working well is a new bug ?

**Comment by frankrousseau** [comment]
Mmm it's embarassing. That bug happened before but if you have two artists with the exact same name, I'm not sure that we can fix this.

**Comment by frankrousseau** [comment]
And without concrete examples, I won't be able to fix this issue

**Comment by angel-k** [comment]
Here's an example:

If I create 2 users Clément Deschamps and I assign them both a task. When I search for the tasks assigned to Clément Deschamps, only those of the 2nd user are displayed.
If I change the name of the 2nd Clément by Clément DESCHAMPS. When I search for the tasks assigned to Clément Deschamps, same situation, only those of the 2nd Clément are displayed.
![case1](https://user-images.githubusercontent.com/25372475/127019723-abade022-4bc7-471c-a5c8-291e73376038.png)

If I change the name of the 2nd Clement to Clement DESCHAMPS. When I search for the tasks assigned to Clement Deschamps, only those of the 1st Clément are displayed.
I notice that when I renamed the 2nd Clément to Clement DESCHAMPS, he changed position in the user list.
![case2](https://user-images.githubusercontent.com/25372475/127019743-a43240a8-72aa-4f14-8d57-7080bb0cb601.png)

If I disable the 2nd Clément, I revert to the initial situation, only the tasks of the 2nd Clément are displayed.
![case3](https://user-images.githubusercontent.com/25372475/127019951-921bfc8f-81da-49fc-93eb-a071e6208b2e.png)

I conclude that Kitsu only displays the tasks of the last user who has this name in the user list.
Thank you !

**Comment by frankrousseau** [comment]
Yes, it's going to be tough to change. We could use the ID instead of the user name but the filter won't be readable at all.

**Comment by angel-k** [comment]
Indeed, using the ID is not necessarily the best solution.
But wouldn't it be possible that the filter displays all occurrences and not only the last one?
